tests/check_consistent.py

```diff
@@ -35,4 +96,6 @@ def main():
 
 
 if __name__ == "__main__":
-    main()
+    check_stdlib()
+    check_stubs()
+    check_same_files()
```

Comment:
Should we also validate METADATA.toml files here?

Reply:
I think yes. I will add a test for it too.